src/current/v25.1/do.md

```diff
@@ -0,0 +1,101 @@
+---
+title: DO
+summary: The DO statement executes a SQL or PL/pgSQL code block.
+toc: true
+keywords:
+docs_area: reference.sql
+---
+
+The `DO` [statement]({% link {{ page.version.version }}/sql-statements.md %}) defines a code block that executes SQL or [PL/pgSQL]({% link {{ page.version.version }}/plpgsql.md %}) syntax.
+
+## Required privileges
+
+- To execute a `DO` block, a user must have at least the [`USAGE` privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) on the schema where the `DO` block is being executed.
```

Comment:
Can you explain this one? I don't think it's true in either CRDB or Postgres.

Reply:
Yeah sorry, that was kind of a guess. I ported it over from the CREATE PROCEDURE and CREATE FUNCTION prereqs. What exactly are the privileges we need on DO blocks? The PG doc says ```The user must have USAGE privilege for the procedural language, or must be a superuser if the language is untrusted. This is the same privilege requirement as for creating a function in the language.```

Reply:
We don't currently do language privileges AFAIK, so I think we should just leave that out for now.